### PR TITLE
do a small simple template update

### PR DESCRIPTION
**Pull request by kafischer**
- add simple secondary process
- simplify the learns about user process

### CODE REVIEW

File: soul/mentalProcesses/shouts.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      - Respond in ALL CAPS
+      - Use capital letters only
+      - Be angry
+      - Be very angry
+    `),
+    { stream: true, model: "quality" }
+  );
+  speak(stream);
+
+  return nextStep
```
Comment (tobowers) [inline soul/mentalProcesses/shouts.ts:19]: so there's no way out of shouts?

**Review by tobowers — APPROVED** [review]
just the one question, but seems good to merge.

File: soul/mentalProcesses/shouts.ts (hunk 1 above)
Comment (kafischer) [reply soul/mentalProcesses/shouts.ts:19]: let me fix

File: soul/mentalProcesses/shouts.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  const lastStep = await nextStep
+  const shouldChill = await lastStep.compute(
+    mentalQuery("The interlocuter apologized")
+  )
+  log("User apologized?", shouldChill)
+  if (shouldChill) {
+    const finalStep = lastStep.withMemory([{
+        role: ChatMessageRoleEnum.Assistant,
+        content: `${initialStep.entityName} thought to themself: I need to chill and stop shouting. I will stop using all caps.`
```
Comment (tobowers) [inline soul/mentalProcesses/shouts.ts:29]: 🤣

**Comment by tobowers** [comment]
nice!

**Pull request merged**